src/formatProcessor.ts

```diff
@@ -320,8 +325,14 @@ export class FormatProcessor extends ConfigPropsProvider {
 					});
 				}
 			);
+ 
+			var result = newText.join('');
```

Comment:
`var` is restricted by eslint, please follow that rules and turn on the linter in VSCode.
```suggestion
			let result = newText.join('');
```

Reply:
Nice! Wasn't aware of this, long time programmer in many languages, but 0 experience with JS 😅 Enabled eslint and of course, it immediately pointed out this line 👍